### DISCUSSION TITLE
Get Threads

### DISCUSSION THREAD

**Discussion by azreal1231** [O365/python-o365#792] [Q&A]
Hi, How would one get threads, so all reply's from both ends of the started email, 
Im assuming it has something to so with the **conversation_id**?

I understand that if i reply to an email via my app, and if open my reply's object it has the body of the email i replied to in it
but going back to the email object i replied to doesn't have the my reply body to it, so i was wondering how do i link them together.

**Comment by alejcas** [comment]
I don't know how that works. Maybe you can check MS Graph docs to see if conversation_id is what you are looking for,

I remember someone also tried this in the past but don't know what was the result.
